post.py

```diff
@@ -79,3 +80,32 @@
 # mpas_analysis tasks
 mpas_analysis(config, scriptDir)
 
+# Create page for the simulation
```

Comment:
Might be useful to add variables for `generate_global_time_series_plots.sh` to the configuration file, so it can be run automatically from `post.py`. Then we can also find the global time series link to display.

Reply:
See #26 for the global time series plot generation